validator/sawtooth_validator/journal/chain.py

```diff
@@ -797,7 +797,7 @@ def _set_genesis(self, block):
                            "genesis block: %s", block)
 
     def _make_receipts(self, results):
-        receipts = {}
+        receipts = []
```

Comment:
This change should be squashed with the previous commit

Reply:
Fixed